app/src/logger/DiscordTransport.ts

```diff
@@ -0,0 +1,76 @@
+import Transport from 'winston-transport';
+import axios, {AxiosInstance, AxiosProxyConfig} from 'axios';
+import {get} from 'lodash';
+
+export default class DiscordTransport extends Transport {
+    private readonly webhookUrl: string;
+    private readonly username?: string;
+    private readonly avatarUrl?: string;
+
+    private readonly axios: AxiosInstance;
+
+    constructor(opts: DiscordTransportOptions) {
+        super(opts);
+
+        this.webhookUrl = opts.webhookUrl;
+        this.username = opts.username;
+        this.avatarUrl = opts.avatarUrl;
+
+        this.axios = axios.create({
+            proxy: opts.proxy ?? undefined,
+        });
+    }
+
+    public log(info: any, callback: () => void): any {
```

Comment:
Suppress any warning

Edit: Also wouldn't this always be a string? Would it handle an error object?

Reply:
No, it is an object and I can't change it as it is defined by winston :\

Reply:
Alright, just suppress it then as it's a 3rd party issue